berkdb/db/db_dispatch.c

```diff
@@ -1290,6 +1290,8 @@ __db_add_limbo(dbenv, info, fileid, pgno, count)
 	if ((ret = __dbreg_id_to_fname(dblp, fileid, 0, &fnp)) != 0)
 		return (ret);
 
+    __ufid_sanity_check(dbenv, fnp);
```

Comment:
Indentation looks off (sorry that roborivers does not check berkdb & sqlite format).